qutebrowser/misc/sessions.py

```diff
@@ -564,3 +597,28 @@ def session_delete(self, name, force=False):
                                       .format(e))
         else:
             log.sessions.debug("Deleted session {}.".format(name))
+
+    @cmdutils.register(instance='session-manager')
+    @cmdutils.argument('name', completion=miscmodels.session)
+    def session_add_url(self, name, force=False):
+        """Add URL to session.
+
+        Args:
+            name: The name of the session.
+            force: Force adding URL to internal sessions (starting with an
+                   underline).
+        """
+        if name.startswith('_') and not force:
+            raise cmdexc.CommandError("{} is an internal session, use --force "
+                                      "to load anyways.".format(name))
+
+        try:
+            self.add_url_to_session(name)
+        except SessionNotFoundError:
+            raise cmdexc.CommandError("Session {} not found!".format(name))
+        except SessionError as e:
+            log.sessions.exception("Error while adding url to session!")
+            raise cmdexc.CommandError("Error while adding url to session: {}"
+                                      .format(e))
+        else:
+            log.sessions.debug("URL added to session {}.".format(name))
```

Comment:
I think this was already logged in `add_url_to_session` right?

Reply:
Removed the other one.